crawlers/mooncrawl/mooncrawl/stats_worker/dashboard.py

```diff
@@ -411,6 +411,60 @@ def get_unique_address(
     )
 
 
+def get_blocks_state(
+    db_session: Session, blockchain_type: AvailableBlockchainType, address: str
+):
+
+    """
+    Generate meta information about
+    """
+
+    blocks_state = {
+        "blocks_state": {
+            "latest_block": {"address": 0, "labels": 0, "transactions": 0},
+            "earliest_block": {"address": 0},
+        }
+    }
+
+    label_model = get_label_model(blockchain_type)
+
+    transactions_model = get_transaction_model(blockchain_type)
+
+    max_transactions_number = db_session.query(
+        func.max(transactions_model.label("block_number"))
+    ).subquery(name="max_transactions_block")
+
+    max_labels_block = db_session.query(
+        func.max(label_model.block_number.label("block_number"))
+    ).subquery(name="max_transactions_block")
+
+    result = (
+        db_session.query(
+            func.min(label_model.block_number),
+            func.max(label_model.block_number),
+            max_transactions_number.c.block_number,
+            max_labels_block.c.block_number,
+        )
+        .filter(label_model.address == address)
```

Comment:
I don't expect to see this filter. Otherwise you are making redundant functionality with `latest_events` on the provider side.

This method is just to get information about crawler status, not about particular events for this address.